pkgs/top-level/all-packages.nix

```diff
@@ -2964,7 +2964,10 @@ with pkgs;
 
   gnu-pw-mgr = callPackage ../tools/security/gnu-pw-mgr { };
 
-  gnused = callPackage ../tools/text/gnused { };
+  gnused = if !stdenv.hostPlatform.isWindows then
```

Comment:
I don't think this is a good idea. Better to patch the latest version than use some old one.

Reply:
The latest version is not so trivial to fix